### PR TITLE
Json servlet updates

### PR DESCRIPTION
**Pull request by pmv**
- add ability to restrict searches to specific projects
- use application/json content type

### CODE REVIEW

File: src/org/opensolaris/opengrok/web/JSONSearchServlet.java [hunk 1]
```diff
@@ -47,6 +47,7 @@
     private static final String PARAM_PATH = "path";
```
Comment (tulinkry) [inline src/org/opensolaris/opengrok/web/JSONSearchServlet.java]: wrong year in copyright

**Comment by tulinkry** [comment]
Except the copyright this is ok to me

**Comment by tarzanek** [comment]
lgtm, if OCA is sorted feel free to merge
thank you!

**Comment by vladak** [comment]
@pmv if you don't have OCA number already, please start the process and state that you agree with the OCA terms conditions so that this PR can be merged.

**Comment by tarzanek** [comment]
@pmv OCA : http://www.oracle.com/technetwork/community/oca-486395.html
if you say you agree with OCA here, we can merge immediatelly (and you can follow up with OCA in parallel), tia
L

**Comment by tarzanek** [comment]
@pmv we plan on releasing very soon, any chance you can get OCA sorted?
I think this would be good to have this PR in

**Comment by pmv** [comment]
Sorry for the delay.  I just received notice that the OCA has been approved from my side, so we should be good to go.

**Comment by vladak** [comment]
Great, thanks !

**Pull request merged**